style.css

```diff
@@ -0,0 +1,188 @@
+body{
+    background-color: black
+}
+
+.container{
+    position: absolute;
+    width: 26rem;
+    left: 50%;
+    top : 10%;
+    margin-left: -13rem;
+    height: 80%;
+    border-style: solid;
+    border-radius: 2%;
+    border-color: white;
+    align-items: center;
+}
+
+.title{
+    width: 90%;
+    margin-left: 5%;
+    height: 38px;
+    color: white;
+    font-size: 27px;
+    text-align: center;
+    font-weight: 800;
+
+    border-bottom: solid;
+    border-radius: 2%;
+}
+
+.dataInput{
+    margin-top: 1.5rem;
+    position: absolute;
+    margin-left: 10%;
+    width:80%;
+    height: 30px;
+    border-style: solid;
+    border-width: 1px;
+    border-color: white;
+    position: relative;
+
+}
+
+#todoText{
+    position: absolute;
+    width: 99%;
+    height: 28px;
+    margin-right: 0.2rem;
+    text-align: center;
+    background-color: black;
+    color: gray;
+    border-style:none; 
+}
+
+#btnAdd{
+    position: relative;
+    margin-left: 90%;
+    background-color: white;
+    border: solid 0.5px white;
+    width: 35px;
+    height: 30px;
+    text-align: center;
+    color: black;
+    font-size: 24px;
+}
+
+.todoBox{
+    position: relative;
+    width: 82%;
+    height: 32%;
+    margin-left: 10%;
+    margin-top: 1rem;
+}
```

Comment:
todoList와 doneList를 렌더링 하는 부분에서 리스트 개수가 너무 많으면 overflow가 발생하여 정해놓은 div를 넘어가는 경우가 있는데요, 이때 `overflow-y: scroll'혹은 `overflow-y: auto`를 사용하면 나름 깔끔하게 스크롤바를 만들 수 있습니다! [스크롤바는 커스터마이징도 가능해요](https://www.w3schools.com/howto/howto_css_custom_scrollbar.asp)

![캡처](https://user-images.githubusercontent.com/77270547/226115502-9d0572d9-441a-49df-a916-4483f79c7374.PNG)

![캡처1](https://user-images.githubusercontent.com/77270547/226115557-03a3678c-85a8-456c-bdfa-23b9b68abcf2.PNG)



```suggestion
.todoBox{
    position: relative;
    width: 82%;
    height: 32%;
    margin-left: 10%;
    margin-top: 1rem;
    overflow-y: auto;
}
```

Reply:
헉 스크롤바도 커스터 마이징이 가능하군요!! 나중에 유용하게 쓸 수 있겠어요!!! 감사합니당👏🏻👏🏻